### ISSUE TITLE
fix gw_drag I/O bugs

### ISSUE THREAD

**Issue by PeterHjortLauritzen** [PeterHjortLauritzen/opt-se-cslam#2]

**Comment by PeterHjortLauritzen** [comment]
When running FW2000 compset with SE I am getting the following error at runtime:

setup_interpolation_and_define_vector_complements: No meridional match for UTGW

Error goes away if I set

fincl1 = 'PS'
fincl2 = 'PS'

I think the problem is in gw_drag.F90 where a number (many) vector components are defined with addfld but not added as vectors ("call register_vector_field" not called). For example, TAUGWX and TAUGWY are vector components but I do not see a call to "register_vector_field". This works fine for CAM-FV since it uses a lat-lon grid but for SE one needs both vector components for I/O.

Here is a version on Cheyenne where I have tried to fix some of the addfld problems (but there are still some questionable fields; marked with "!phl vector???"). All changes are marked with "!phl":

~pel/src/trunk-aug15/components/cam/src/physics/cam/gw_drag.F90

FYI: the model still crashes with

FLDLST: TAQ in fincl(736, 1) not found
 FLDLST: THzm in fincl(755, 1) not found
 FLDLST: UVzm in fincl(785, 1) not found
 FLDLST: UWzm in fincl(786, 1) not found
 FLDLST: Uzm in fincl(787, 1) not found
 FLDLST: VTHzm in fincl(791, 1) not found
 FLDLST: Vzm in fincl(793, 1) not found
 FLDLST: Wzm in fincl(873, 1) not found

which I believe are outfid calls from the FV core: zonal means (which we can't do in SE). Is there a way for the use_cases to be dycore dependent?
